frontend/packages/network-attachment-definition-plugin/src/components/network-attachment-definitions/types.ts

```diff
@@ -0,0 +1,20 @@
+import { NetworkAttachmentDefinitionConfig } from '../../types';
+
+export type NetAttachDefBundle = {
+  name?: string;
+  namespace?: string;
+  type?: string;
+  metadata?: any;
+  configJSON?: NetworkAttachmentDefinitionConfig;
+  netAttachDef: any;
+};
+export type NetworkAttachmentDefinitionsRowProps = {
```

Comment:
Nit: please separate type declarations with newline.

Reply:
Done! I don't know how I missed that. I can't stand when everything is punched together like that!